src/main/features/claim/views/amount.njk

```diff
@@ -63,17 +66,17 @@
                   {% endif %}
                   {{
                     textInputInline(
-                      'rows['+loop.index0+'][reason]', form, 'claim-amount-column-1' + (' form-control-error' if displayTotalError else ''),
-                      ariaLabelledBy = "claim-reason-label",
+                      reasonJoinKey, form, 'claim-amount-column-1' + (' form-control-error' if displayTotalError else ''),
+                      arialabelledby = "claim-reason-label",
                       ariaDescribedBy = "claim-reason-hint"
                     )
                   }}
-
+                  <label class="form-label-bold mobile-show visually-hidden" for="{{ reasonJoinKey }}">{{ possition }} {{ t('. What you’re claiming for') }}</label>
                   <span class="form-label-bold mobile-show">{{ t('Amount') }}</span>
+                  <label class="form-label-bold mobile-show visually-hidden" for="{{ amountJoinKey }}">{{ possition }} {{ t('. Amount') }}</label>
                   {{
                     textInputInline(
```

Comment:
'textInputInline' macro must have label in its macro. Can you check if that can be used?

Reply:
Updated the macro, So that we have single label for the input field.
![image](https://user-images.githubusercontent.com/41957682/91040365-01a4ef00-e606-11ea-9ac7-7d3416754309.png)